src/hashTable.js

```diff
@@ -0,0 +1,148 @@
+'use strict'
+
+const ARBITRARY_ZIP_CODE_HASH = 94607
+const LENGTH_OF_ALPHABET = 26
+const node = ( key, value ) => {
+
+  let obj = {}
+  obj[ key ] = value
+  return obj
+
+	}
+
+export default class HashTable {
```

Comment:
I refuse to review the  code below this line. The formatting below is riddled with violations. Please clean this up and re-submit for review.